src/utils/analytics.ts

```diff
@@ -0,0 +1,26 @@
+import { Analytics, TrackParams } from '@segment/analytics-node';
+
+const analytics = new Analytics({
+  writeKey: process.env.SEGMENT_WRITE_KEY || 'cwWhBWso2gwzyenW29CnCp30Kjz4DogE',
```

Comment:
What account is this default key for? Do we even need to expose a default here?

Reply:
@tevanoff yes, this is a new source for this addon. For segment in client side code you need to expose the write keys, that's just how it works 🤷 

Do you think I should locate the key somewhere else? It'll need to be built into the npm package somewhere.

Reply:
Oh right! I was thinking of pulling the value in from an env var at build time but that doesn't make sense here.